src/backend/distributed/operations/node_protocol.c

```diff
@@ -576,22 +576,6 @@ GetPreLoadTableCreationCommands(Oid relationId,
 
 	PushOverrideEmptySearchPath(CurrentMemoryContext);
 
-	/* if foreign table, fetch extension and server definitions */
-	char tableType = get_rel_relkind(relationId);
```

Comment:
Just noting for future reference. Our reason to remove these here has two aspects:

 (a) we do not find dependency between the server and the extension as we expect all extensions to be already distributed and exists on the workers
(b) The server has already been created and exists on the workers